.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+### Describe your changes
+
+## Issue ticket number and link
+
+### Checklist before requesting a review
+#### General
+- [ ] I have performed a self-review of my code
+- [ ] The commit message follows our guidelines
+- [ ] I have put the PR link into the planner
+- [ ] The PR has a clear and descriptive title and a detailed description of the changes
+
+#### Code Quality
+- [ ] Code Style
+- [ ] Linting
+- [ ] Documentation
+- [ ] Comments
+- [ ] No Console Logs/Debuggers
+
+#### Testing
+- [ ] Unit tests have been added or updated to cover the changes
+- [ ] Integration tests have been added or updated where necessary
+- [ ] The changes have been tested manually in a local environment
+
+#### Deployment
+- [ ] All necessary environment variables have been documented and added to the deployment environment
+- [ ] Any required database migrations have been added and tested
```

Comment:
I would condense this to something like "documentation has been updated where needed" maybe there is another one for DBs. The one I see mainly is updating existing migration files, don't do that